src/skodun/dispatch.py

```
@@ -1178,13 +1179,25 @@ def run_worker(record_id: str, repo: Path, branch: str, local_oid: str,
                 return WorkerOutcome(0, _banner_failure(reason))
+            finally:
+                owner = owner_context.get('owner')
+                if owner is not None:
+                    try:
+                        current = store.get_review(record_id) or {}
+                        if owner.matches(current) and current.get('status') != 'running':
+                            from .control import cancellation_completion
+                            from .requests import now
+                            store.finish_cancellations(target_id=record_id,
+                                outcome=cancellation_completion(current), now=now())
+                    except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)